### PR TITLE
P0/auth

### PR DESCRIPTION
**Pull request by purvasharmadev**
## Added authentication

- user can login/logout
- user can signIn

### CODE REVIEW

File: src/Auth/Login/login.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+          </button>
+
+          <h4>
+            <Link to="/signUp" className="link color-primary">
+              Create A New Account <i className="fa fa-arrow-right fa-x"></i>
+            </Link>
+          </h4>
+        </form>
+      </div>
+      {/* <div>
```
Comment (tonymike99) [inline src/Auth/Login/login.js:102]: You can remove these comments

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,13 @@
 import "./App.css";
-import URLRoutes from "./routes"
+import URLRoutes from "./routes";
 
 function App() {
-  const apiKey = process.env.REACT_APP_JWT_SECRET
-  console.log(apiKey)
+  const apiKey = process.env.REACT_APP_JWT_SECRET;
+  console.log(apiKey);
```
Comment (tonymike99) [inline src/App.js:6]: Avoid logging the apiKey.. It is bad practice.

File: src/Pages/Home/home-page.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  text-align: center;
+}
+
+.Home-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+.Home-header {
+  /* background-color: #22201c; */
```
Comment (tonymike99) [inline src/Pages/Home/home-page.css:11]: Remove comment

**Review by tonymike99 — APPROVED** [review]
Hi Purva, 
The code looks fine. 
The deploy link is not working. I guess you forgot to set the env variable in netlify.
Suggestion: Ensure that you remove unnecessary comments. And don't use console logs (except for errors) especially for the api key. That is bad practice.

**Review by adityapandadev — APPROVED** [review]

**Comment by purvasharmadev** [comment]
> Hi Purva, The code looks fine. The deploy link is not working. I guess you forgot to set the env variable in netlify. Suggestion: Ensure that you remove unnecessary comments. And don't use console logs (except for errors) especially for the api key. That is bad practice.

Thank you for your reviews, will fix the mistakes.

**Pull request merged**